Sources/StreamVideoSwiftUI/Utils/PictureInPicture/StreamPictureInPictureVideoRenderer.swift

```diff
@@ -8,7 +8,7 @@ import StreamVideo
 import StreamWebRTC
 
 /// A view that can be used to render an instance of `RTCVideoTrack`
-final class StreamPictureInPictureVideoRenderer: UIView, RTCVideoRenderer {
```

Comment:
Why removing the conformance if it doesn't require marking as @preconcurrency?

Reply:
leftover, fixed